data-science-onramp/data-cleaning/clean.sh

```diff
@@ -0,0 +1,12 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
+# Requires having PROJECT_ID, CLUSTER_NAME and BUCKET_NAME set as 
+# environment variables
+
+export CLUSTER_NAME=data-cleaning
+export PROJECT_ID=data-science-onramp
+export BUCKET_NAME=citibikevd
+
+gcloud dataproc jobs submit pyspark --cluster ${CLUSTER_NAME} \
+    --jars gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar \
+    --driver-log-levels root=FATAL \
```

Comment:
What does this flag do, and why do we have it set to the value it is set to?

Reply:
This flag sets the logging level of the job. The special root package controls the root logger level. It is set to FATAL, which is described as "severe errors that cause premature termination."

I think a lower logging level (like WARN for instance) might perhaps produce too much verbose output, especially for the scope of the tutorial. What do you think?